gulpfile.js

```diff
@@ -36,7 +36,9 @@ global.cliOptions = options;
 
 // Forward referencing means the order of gulp-task
 // requires is important
-const gulpTaskFiles = [
+const exportedTasks = [
```

Comment:
This could presumably just glob for files and require them all.

Reply:
Yeah, with the move to functions instead of strings, the order tasks are registered wouldn't matter.  I really like `export default as "build" from './gulp-tasks/build'` though - might be easier to convert later if these were `exports.build = require('./gulp-tasks/build')`. This would require the top-level tasks to avoid `-` or `:` in their names.